MFVNeutralino/test/TriggerStudies/eff.py

```diff
@@ -129,12 +129,15 @@
     from JMTucker.Tools.MetaSubmitter import *
 
     if year == 2017:
-        samples = Samples.leptonic_samples_2017 + Samples.met_samples_2017
+        #samples = Samples.leptonic_samples_2017 + Samples.met_samples_2017
+        samples = [getattr(Samples, 'wjetstolnu_2j_2017')]
+        """
         if useElectron:
           samples += Samples.singleelectron_data_samples_2017
         else:
           samples += Samples.auxiliary_data_samples_2017
         samples += Samples.mfv_splitSUSY_samples_2017
+        """
```

Comment:
Without looking too closely, I am surprised that the default that we're pushing here is to only run on one MC sample, rather than the full suite of samples that was there before?

Reply:
Hmm, is this for the lepton trigger efficiency measurement via the MET trigger, or something else? it's not immediately obvious. We ultimately need to run that over data, of course.